app/Listeners/SendTelegramNewArticleNotification.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Listeners;
+
+use App\Events\ArticleWasSubmittedForApproval;
+use App\Notifications\ArticleSubmitted;
+use Illuminate\Notifications\AnonymousNotifiable;
+
+final class SendTelegramNewArticleNotification
```

Comment:
```suggestion
final class SendNewArticleNotification
```

The notification channel is an implementation detail and doesn't needs to be specified here. For example, later on, we could change or add other notification channels when necessary.

Reply:
Done